core/workspace.js

```diff
@@ -197,91 +209,7 @@ Blockly.Workspace.prototype.clear = function() {
     Blockly.Events.setGroup(false);
   }
   this.variableMap_.clear();
-};
-
-/**
- * Walk the workspace and update the map of variables to only contain ones in
- * use on the workspace.  Use when loading new workspaces from disk.
- * @param {boolean} clear True if the old variable map should be cleared.
- */
-Blockly.Workspace.prototype.updateVariableStore = function(clear) {
-  // TODO: Sort
-  if (this.isFlyout) {
-    return;
-  }
-  var variableNames = Blockly.Variables.allUsedVariables(this);
-  var varList = [];
-  for (var i = 0, name; name = variableNames[i]; i++) {
-    // Get variable model with the used variable name.
-    var tempVar = this.getVariable(name);
-    if (tempVar) {
-      varList.push({'name': tempVar.name, 'type': tempVar.type,
-          'id': tempVar.getId()});
-    } else {
-      varList.push({'name': name, 'type': null, 'id': null});
-      // TODO(marisaleung): Use variable.type and variable.getId() once variable
-      // instances are storing more than just name.
-    }
-  }
-  if (clear) {
-    this.variableMap_.clear();
-  }
-  // Update the list in place so that the flyout's references stay correct.
-  for (var i = 0, varDict; varDict = varList[i]; i++) {
-    if (!this.getVariable(varDict.name)) {
-      this.createVariable(varDict.name, varDict.type, varDict.id);
-    }
-  }
-};
-
-/**
- * Rename a variable by updating its name in the variable map. Identify the
- * variable to rename with the given variable.
- * @param {?Blockly.VariableModel} variable Variable to rename.
- * @param {string} newName New variable name.
- * @param {string=} opt_type The type of the variable to create if variable was
- *     null.
- */
-Blockly.Workspace.prototype.renameVariableInternal_ = function(
-    variable, newName, opt_type) {
-  var type = variable ? variable.type : (opt_type || '');
-  var newVariable = this.getVariable(newName, type);
-  var oldCase;
-
-  // Find if newVariable case is different.
-  if (newVariable && newVariable.name != newName) {
-    oldCase = newVariable.name;
-  }
-
-  Blockly.Events.setGroup(true);
-  var blocks = this.getAllBlocks();
-  // Iterate through every block and update name.
-  for (var i = 0; i < blocks.length; i++) {
-    blocks[i].renameVar(variable.name, newName);
-    if (oldCase) {
-      blocks[i].renameVar(oldCase, newName);
-    }
-  }
-  this.variableMap_.renameVariable(variable, newName, type);
-  Blockly.Events.setGroup(false);
-};
-
-
-/**
- * Rename a variable by updating its name in the variable map. Identify the
- * variable to rename with the given name.
- * TODO (#1199): Possibly delete this function.
- * @param {string} oldName Variable to rename.
- * @param {string} newName New variable name.
- * @param {string=} opt_type The type of the variable.  If not provided it
- *     defaults to the empty string, which is a specific type.
- */
-Blockly.Workspace.prototype.renameVariable = function(oldName, newName,
```

Comment:
Should we delete these APIs or do a lookup by name and type and call the new one with the variable?